src/views/Player.vue

```diff
@@ -97,7 +100,8 @@ export default {
           process.env.VUE_APP_BACKEND +
             process.env.VUE_APP_BACKEND_PLIO_DETAILS +
             "?plioId=" +
-            this.$route.params.id
+            this.$route.params.id +
+            "&userId=" + this.studentId
```

Comment:
Why this mismatch between userID and studentID? make it userID only everywhere no?